.env.sample

```
@@ -6,14 +6,18 @@
 # =============================================================================
 QDRANT_URL=https://your-cluster.qdrant.io:6333
 QDRANT_API_KEY=your-qdrant-api-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The QDRANT_API_KEY key should go before the QDRANT_URL key

(UnorderedKey)